### PR TITLE
NAS-100895: Reorganize network.rst and add initial text review

### PR DESCRIPTION
**Pull request by Mrt134**
Add intro entry for major rework to networking.
Reorganize network.rst: Interfaces now contains all text related to Link Aggregations VLANs. A new subsection is added for network bridges.
Rewrite Interfaces section to reflect changes made to the new UI:
 - All interfaces are listed
 - All configuration options for different interface types
 - Expanded descriptions for list items in Network > Interfaces

Write initial blob for Bridges subsection
Begin reviewing link aggregation and VLAN section text.
Remove several obsolete images.
HTML build testing: no issues.

Continued review is required and is hopefully tracked in other Jira tickets.

### CODE REVIEW

**Comment by Mrt134** [comment]
Investigation - none of these changes are visible in the legacy UI as of the FreeNAS-11.3-MASTER-201906250622 nightly.

File: userguide/network.rst [hunk 1]
```diff
@@ -170,272 +169,153 @@ Interfaces
...
-
+   +---------------------+----------------+-------------+-----------------------------------------------------------------------------------------------------------+
+   | Setting             | Value          | Type        | Description                                                                                               |
+   |                     |                |             |                                                                                                           |
+   +=====================+================+=============+===========================================================================================================+
+   | Type                | drop-down menu | All         | Choose the type of interface. *Bridge* creates a logical link between multiple networks.                  |
+   |                     |                |             | *Link Aggregation* combines multiple network connections into a single interface. A virtual LAN (*VLAN*)  |
+   |                     |                |             | partitions and isolates a segment of the connection.                                                      |
```
Comment (a-stjohn) [inline userguide/network.rst]:
```suggestion
   |                     |                |             | partition and isolates a segment of the connection.                                                       |
```

Comment (a-stjohn) [reply userguide/network.rst]: Woops. I must've over looked a piece of that sentence. Disregard this suggestion. The sentence makes sense.

File: userguide/intro.rst [hunk 2]
```diff
@@ -218,9 +228,14 @@ These screen options have changed:
   :menuselection:`Network --> Interfaces --> ADD` has been renamed
   to :guilabel:`Description`.
 
+<<<<<<< HEAD
```
Comment (wblock) [inline userguide/intro.rst]: Some leftover merge text still here.

File: userguide/intro.rst [hunk 3]
```diff
@@ -218,9 +228,14 @@ These screen options have changed:
...
+* The :guilabel:`MTU` field has been added to
+  :menuselection:`Network --> Interfaces --> ADD`.
+=======
```
Comment (wblock) [inline userguide/intro.rst]: And here.

File: userguide/intro.rst [hunk 4]
```diff
@@ -218,9 +228,14 @@ These screen options have changed:
...
 * A :ref:`dataset <Adding Datasets>` deletion confirmation screen has
   been added to
   :menuselection:`Storage --> Pools --> Delete Dataset`.
+>>>>>>> fc2d212e78f330a31f3ec65dd396e6bb453a5b0a
```
Comment (wblock) [inline userguide/intro.rst]: Here too.

**Pull request merged**